src/main/scala/org/broadinstitute/dsde/firecloud/utils/StreamingPassthrough.scala

```diff
@@ -114,6 +113,25 @@ trait StreamingPassthrough
     req.withUri(targetUri).withHeaders(targetHeaders)
   }
 
+  // TODO: get feedback on this method. Not happy having rawls specific request handling here (rather have it on CromiamApiService)
+  /**
+   * Private method that modifies the requestPath for the one workflow passthrough to Rawls
+   * due to the fact that the Rawls endpoint has a different URI construction, making suffix
+   * passthroughs inept
+   * @param requestUri
+   * @param remoteBaseUri
+   * @return the requestPath (String) suitable to be sent to the remote system
+   */
+  private def requestStringConstruction(requestUri: Uri, remoteBaseUri: Uri) : String = {
+    val requestPath = requestUri.path.toString
+    if (
+        FireCloudConfig.Rawls.authUrl.contains(remoteBaseUri.authority.toString()) &&
+        requestPath.contains("/backend/metadata/")
+    ) {
+      requestPath.replace("backend/metadata", "genomics")
+    } else requestPath
+  }
```

Comment:
It looks like this replaces the passthrough to CromIAM's metadata endpoint with a passthrough to the Rawls genomics endpoint. I think we need to retain the endpoint that goes to CromIAM's metadata endpoint, I expect that's used by Job Manager to get all the other information to display, and maybe some other things as well. 

I think we want to support both the CromIAM metadata passthrough, and a Rawls genomics passthrough.

Reply:
I don't know what you mean by CromIAM endpoint. Do you mean `api/workflows/{version}/{id}/metadata`? If so, it works locally for me
<img width="1789" alt="Screen Shot 2023-01-13 at 2 22 38 PM" src="https://user-images.githubusercontent.com/6664778/212401944-5c4a6bac-5f2f-43f8-a757-13518ccafe33.png">

If not, then I'm not sure. CromIAM does not have an endpoint that follows the structure `{version}/{id}/backend/metadata/{operationId}`
<img width="1693" alt="Screen Shot 2023-01-13 at 2 38 12 PM" src="https://user-images.githubusercontent.com/6664778/212404740-c03781fa-b86b-4ef5-a090-53854b6719f2.png">



To Adam's point, this is why I don't like my implementation, but I'm not sure how to tackle the path modification. The `streamingPassthrough (unaltered)` method will always access the path value directly from the request object and appends it to the remoteUrl, which does not work for Rawls since its endpoint is structured differently. 

At the same time, I don't want to change `streamingPassthrough's` core functionality since it is valid for every other route. I would like to access the request object at a higher level (service ideally, its where the route matching happens), I just don't know how.